docs/standards/nft-2.0/LSP8-Identifiable-Digital-Asset.md

```diff
@@ -13,83 +13,105 @@ sidebar_position: 4
 
-#### 1. The metadata attached to the NFT (represented with `tokenURI`) refers to a file.
+![LSP8IdentifiableDigitalAsset features Introduction](/img/standards/lsp8-intro.jpeg)
 
-The reference raises several questions around where the token's metadata is stored. Is a central service used to store the metadata? What about if such a service goes down?
+## What does this Standard represent?
 
-![ERC721 Metadata Security](/img/erc721-metadata-security.png)
+### Specification
 
-#### 2. The only data that can be attached to an NFT is a `name` and `symbol`.
+**[LSP8-IdentifiableDigitalAsset](#)** is a standard that aims to describe _non-fungible_ assets. The term _Non-fungible_ means that each asset is **unique and different**. They are distinguishable from each other and therefore **not interchangeable**.
 
-Each NFT has its own characteristics and specific details. The singularity is what makes each NFT unique. But how can we attach more information specific to an NFT? Or **how can we query one precise information** specific to an NFT without having to retrieve and search through the entire metadata?
+This standard was based on **[ERC721](https://eips.ethereum.org/EIPS/eip-20)** and **[ERC1155](https://eips.ethereum.org/EIPS/eip-777)** with additional features mentioned below:
 
-#### 3. The metadata file can be altered.
 
-Suppose the metadata for an NFT is linked to a JSON file. How do we know if it has been altered? The uncertainty presents a significant problem. If the rules and logic of an application were based on the metadata related to an NFT, any unexpected change to the data could affect the whole application.
+### Bytes32 TokenId
 
-#### 4. The NFT metadata is set in stone.
+The current NFT standards such as **[ERC721](https://eips.ethereum.org/EIPS/eip-721)** and **[ERC1155](https://eips.ethereum.org/EIPS/eip-1155)** **lack asset representation** as they define the tokenIds **as Numbers** `(uint256)`. Each token from the NFT collection will be defined and queried based on this tokenId, which is normally incremental.
 
-How about if we want to represent NFTs as more than just a set of information in a JSON file? If we wish NFTs to act as "digital liveable things" that can **change**, **evolve** or **be upgraded over time**?
+![ERC721 TokenIds Representation](/img/standards/erc721-tokenIds.jpeg)
 
-A good example where all these problems could have effects is a video game. Game services could use an NFT to represent a unique item like a weapon. We might want the player to have the ability to upgrade the weapon features over time so that it becomes more powerful.
+**[LSP8-IdentifiableDigitalAsset](#)** define the tokenIds as **bytes32**. The choice of **bytes32 tokenIds** allows a wide variety of applications including numbers, contract addresses, and hashed values (ie. serial numbers). 
 
-## What does this Standard represent?
+The **bytes32 tokenId** can be interpreted as a <u>**Number**</u>: 
```

Comment:
How about doing 3 bullet points, and all the figures below? (to avoid sentence repetition _"the bytes32 tokenId can be interpreted as ..."_